File: backend/ms_sms/src/main/java/wevioo/tn/ms_sms/dtos/request/SendsSms.java
package wevioo.tn.ms_sms.dtos.request;

import lombok.Data;

import java.util.List;
import java.util.Map;
@Data
public class SendsSms {
    private Long idTemplate;
    private List<String> numbers;
    private Map<String, String> placeholderValues;
}

File: frontend/src/models/sms/SendsSms.ts
export interface SendsSms {
    idTemplate: number;
    numbers: string[];
    placeholderValues: { [key: string]: string };
}
export interface SendSeperately {
    number: string;
    placeholderValues: { [key: string]: string };
}
export interface SendIndiv {
    idTemplate: number;
    sendSeparatelyList:SendSeperately[]
}
